nano/scripts/nano/nano_template.js

```diff
@@ -5,19 +5,19 @@ var NanoTemplate = function () {
 
     var _templates = {};
     var _compiledTemplates = {};
-	
-	var _helpers = {};
+    
+    var _helpers = {};
 
     var init = function () {
         // We store templateData in the body tag, it's as good a place as any
-		_templateData = $('body').data('templateData');
+        _templateData = $('body').data('templateData');
 
-		if (_templateData == null)
-		{
-			alert('Error: Template data did not load correctly.');
-		}
+        if (_templateData == null)
+        {
+            reportError('Error: Template data did not load correctly.');
```

Comment:
Writing something invalid like `{{:(}hello` should spam constant nanoui errors in a loop.